apps/trading/components/rewards-container/active-rewards.tsx

```diff
@@ -57,6 +62,30 @@ export type Filter = {
   searchTerm: string;
 };
 
+/** Eligibility requirements for rewards */
+export type Requirements = {
+  isEligible: boolean;
+  stakeAvailable?: bigint;
+  team?: Partial<Team>;
+  pubKey: string;
+};
+
+const Tick = () => (
+  <VegaIcon
+    name={VegaIconNames.TICK}
+    size={18}
```

Comment:
Can we make this size 10 or 12. 18 looks way to big

Reply:
sure, this is 12

![image](https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/2446417a-1e67-4cba-ba48-7b3cc67d7526)